static/install.sh

```diff
@@ -72,3 +73,4 @@ else
 fi
 
 echo "Successfully installed! Now you can run it via invoking $RUNPATH"
+echo "To add opta to your path, please add" 'export PATH=$PATH:'"$RUNPATH" 'to you cli profile'
```

Comment:
I see this even on mac. We should only show this if RUNPATH != opta?

Reply:
easy check to add, right?

Reply:
pushed a change to limit this  path setting suggestion to only Linux